pyproject.toml

```diff
@@ -20,15 +20,21 @@ requires-python = ">3.7"
 requires = [
     'click',
     'eth-utils',
+    'eth-utils',
+    'flask==1.1.2',
```

Comment:
why pin this?

Reply:
Why have it at all is also a good question. I think I may have botched the rebase for the dependencies here.
There is no longer an API server etc. in the SP...